lib/docker-sync/dependencies/package_managers/none.rb

```diff
@@ -0,0 +1,23 @@
+module DockerSync
+  module Dependencies
+    module PackageManager
+      class None < Base
+        NO_PACKAGE_MANAGER_AVAILABLE = 'No package manager was found. Please either install one of those supported (brew, apt, rpm) or install all dependencies manually.'.freeze
+
+        def self.available?
+          true
+        end
+
+        def self.ensure!
+          # noop
+        end
+
+        private
+
+        def install_cmd
+          raise(NO_PACKAGE_MANAGER_AVAILABLE)
```

Comment:
It's supposed to have been output the line before: https://github.com/EugenMayer/docker-sync/pull/391/files#diff-daa72dbe0caf7c551d89ad678f07c384R25